smarts/core/utils/sumo.py

```diff
@@ -47,9 +47,13 @@
 if tools_path not in sys.path:
     sys.path.append(tools_path)
 
-
-import sumo.tools.sumolib as sumolib
-import sumo.tools.traci as traci
+try:
+    import sumo.tools.sumolib as sumolib
+    import sumo.tools.traci as traci
+except ModuleNotFoundError as e:
+    raise ImportError(
+        "Missing dependencies for SUMO. Install them using the command `pip install -e .[sumo]` at the source directory."
+    ) from e
```

Comment:
I can note that all versions of the `"hiway"` environment import `SumoTrafficSimulation` as well as `traffic_histories_to_observations.py`.

Please update those appropriately.

Reply:
SUMO is now optional for these environments, though they still have optional params that are SUMO-specific that I did not touch.